src/openapi.yaml

```diff
@@ -115,21 +115,6 @@ paths:
               - Initiator
               - Klantcontacter
               - Zaakcoördinator
```

Comment:
"Mede-initiator" hier dan toevoegen?

Overigens net gelezen dat de enige case waarvoor dit geldt, is een huwelijk.